README.md

```diff
@@ -0,0 +1,66 @@
+# slog, a back-end agnostic interface for structured logs
+
+[![Go Reference](https://pkg.go.dev/badge/github.com/darvaza-proxy/slog.svg)](https://pkg.go.dev/github.com/darvaza-proxy/slog)
+
+[`slog.Logger`](#interface) provides a simple standardised interface for structured logs for libraries. It supports [six log levels](#log-levels) and [fields](#fields) with unique string labels (keys).
+
+## Interface
+Every method of this interface, with the exception of [`Print()`](#print), returns a `slog.Logger` so it can be daisy chained when composing a log entry.
+A log entry begins setting the [level](#log-levels) followed by optional addition of [fields](#fields) and a [call stack](#call-stack) and ends with a message calling a [Print](#print) method.
```

Comment:
A log entry begins with setting the [level](#log-levels) which is followed by ....